.changeset/clean-chicken-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Use /\_svelte_kit_assets when serving apps with specified paths.assets locally
```

Comment:
does the underscore need to be escaped here?

Reply:
that's just something `pnpx changeset` does. it's harmless